src/Native/Unix/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -1,127 +0,0 @@
-
-project(System.Security.Cryptography.Native)
-
-set(CMAKE_INCLUDE_CURRENT_DIR ON)
-
-# These are happening inside of OpenSSL-defined macros out of our control
-add_compile_options(-Wno-cast-align)
-
-add_definitions(-DPIC=1)
-
-if(CMAKE_STATIC_LIB_LINK)
-   set(CMAKE_FIND_LIBRARY_SUFFIXES .a)
-endif(CMAKE_STATIC_LIB_LINK)
-
-find_package(OpenSSL REQUIRED)
-include_directories(SYSTEM ${OPENSSL_INCLUDE_DIR})
-
-set(NATIVECRYPTO_SOURCES
-    openssl.cpp
-    pal_asn1.cpp
-    pal_asn1_print.cpp
-    pal_bignum.cpp
-    pal_bio.cpp
-    pal_dsa.cpp
-    pal_ecdsa.cpp
-    pal_ecc_import_export.cpp
-    pal_eckey.cpp
-    pal_err.cpp
-    pal_evp.cpp
-    pal_evp_pkey.cpp
-    pal_evp_pkey_dsa.cpp
-    pal_evp_pkey_eckey.cpp
-    pal_evp_pkey_rsa.cpp
-    pal_evp_cipher.cpp
-    pal_hmac.cpp
-    pal_pkcs12.cpp
-    pal_pkcs7.cpp
-    pal_rsa.cpp
-    pal_ssl.cpp
-    pal_x509.cpp
-    pal_x509_name.cpp
-    pal_x509_root.cpp
-    pal_x509ext.cpp
-)
-
-if (FEATURE_DISTRO_AGNOSTIC_SSL)
-    if (NOT CMAKE_SYSTEM_NAME STREQUAL Linux)
-        message(FATAL_ERROR "FEATURE_DISTRO_AGNOSTIC_SSL can only be enabled for Linux")
-    endif()
-
-    list(APPEND NATIVECRYPTO_SOURCES
-        opensslshim.cpp
-    )
-    add_definitions(-DFEATURE_DISTRO_AGNOSTIC_SSL)
-endif()
-
-add_library(objlib OBJECT ${NATIVECRYPTO_SOURCES} ${VERSION_FILE_PATH})
-
-add_library(System.Security.Cryptography.Native
-    SHARED
-    $<TARGET_OBJECTS:objlib>
-)
-
-add_library(System.Security.Cryptography.Native.OpenSsl
-    SHARED
-    $<TARGET_OBJECTS:objlib>
-)
-
-# Disable the "lib" prefix.
-set_target_properties(System.Security.Cryptography.Native PROPERTIES PREFIX "")
-set_target_properties(System.Security.Cryptography.Native.OpenSsl PROPERTIES PREFIX "")
-
-if (FEATURE_DISTRO_AGNOSTIC_SSL)
-    add_custom_command(TARGET System.Security.Cryptography.Native.OpenSsl POST_BUILD
-        COMMENT "Verifying System.Security.Cryptography.Native.OpenSsl.so dependencies"
-        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/../verify-so.sh 
-            $<TARGET_FILE:System.Security.Cryptography.Native.OpenSsl> 
-            "Verification failed. System.Security.Cryptography.Native.OpenSsl.so has undefined dependencies. These are likely OpenSSL APIs that need to be added to opensslshim.h"
-        VERBATIM
-    )
-
-    # Link with libdl.so to get the dlopen / dlsym / dlclose
-    target_link_libraries(System.Security.Cryptography.Native
```

Comment:
All that should be deleted is this (and the custom command for Apple).

The rest of the library is absolutely still in use (just now called System.Security.Cryptography.Native.OpenSsl... the export name from 3 lines down)